packages/server/graphql/intranetSchema/mutations/sendUpcomingInvoiceEmails.ts

```diff
@@ -113,7 +113,9 @@ const sendUpcomingInvoiceEmails = {
       (prev, cur) => prev.concat(cur.billingLeaderIds, cur.newUserIds),
       [] as string[]
     )
-    const allUsers = await getUsersByIds(allUserIds)
+    const allUsers = await Promise.all(
+      allUserIds.map((userId) => dataLoader.get('users').load(userId))
+    )
```

Comment:
why not `const allUsers = await dataLoader.get('users').loadMany(allUserIds)`?

Reply:
`loadMany` returns `(IUser | Error)[]` and needs additional filtering/throwing, while `Promise.all(...load(...))` returns `IUser[]` and throws if there is an issue. Otherwise these are identical since `loadMany` does internally just `Promise.all(...load(...))`.